app/assets/stylesheets/_subnav.scss

```diff
@@ -26,3 +26,86 @@
     }
   }
 }
+
+.study-subnav {
+  position: relative;
+  background-color: $color-off-white;
+  @include gradient-horizontal(
+    $start-color: $color-pale-green,
+    $end-color: $color-off-white,
+    $start-percent: 50%,
+    $end-percent: 50%
+  );
+  color: #aaa;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.